apps/nextjs/src/components/match/add/add-player-form.tsx

```
@@ -0,0 +1,252 @@
+      </DialogHeader>
+      <form
+        id={form.formId}
+        onSubmit={async (e) => {
+          e.preventDefault();
+          await form.handleSubmit();
+        }}
+      >
+        <FieldGroup>
+          <form.Field
+            name="name"
+            children={(field) => {
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)